index.js

```diff
@@ -80,14 +87,15 @@ class MergedStream extends PassThroughStream {
 
 const onMergedStreamFinished = async (passThroughStream, streams) => {
 	updateMaxListeners(passThroughStream, PASSTHROUGH_LISTENERS_COUNT);
-	const abortController = new AbortController();
+	const controller = new AbortController();
+
 	try {
 		await Promise.race([
-			onMergedStreamEnd(passThroughStream, abortController),
```

Comment:
Renamed to `controller` to avoid confusion with streams "abort", which are unrelated but share the same name.